.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -169,13 +169,19 @@ jobs:
         cmake --build $GITHUB_WORKSPACE/build --target utils/not/install
         cmake --build $GITHUB_WORKSPACE/build --target utils/lit/install
         cmake --build $GITHUB_WORKSPACE/build --target utils/llvm-lit/install
-        cmake --build $GITHUB_WORKSPACE/build --target install-clang-format
-        cmake --build $GITHUB_WORKSPACE/build --target install-clang-tidy
         cmake --build $GITHUB_WORKSPACE/build --target install-llvm-size
         cmake --build $GITHUB_WORKSPACE/build --target install-llvm-cov
         cmake --build $GITHUB_WORKSPACE/build --target install-llvm-profdata
         cmake --build $GITHUB_WORKSPACE/build --target install-compiler-rt
 
+    - name: Install clang-format/clang-tidy
+      # We install these into our nightly container that is later used in CI to
+      # run lint task.
+      run: |
+        cmake --build $GITHUB_WORKSPACE/build --target install-clang-format
+        cmake --build $GITHUB_WORKSPACE/build --target install-clang-tidy
```

Comment:
```
        cmake --build $GITHUB_WORKSPACE/build --target install-clang-format install-clang-tidy
```
What do you think about this notation? (It should work if I understand CMake documentation correctly).

We can also try multi-line notation (which seems easier to read?), but I'm not sure if it work.
```
        cmake --build $GITHUB_WORKSPACE/build --target install-clang-format\
                                                       install-clang-tidy
```

Reply:
Later, once I move those into a separate action. I think it might be just an inputs parameters that would get expanded.

Reply:
Okay. I hope this idea won't be lost and you are tracking it in your personal TODO list.